app/assets/stylesheets/forever_style_guide/modules/_modal.scss

```diff
@@ -2,7 +2,8 @@ $modal-padding-footer: 15px;
 $modal-form-width-lg: 355px;
 $modal-product-suggestion-sm: 170px;
 $modal-promo-height: 475px;
-$border-radius-modal: 6px;
+$border-radius-modal: 4px;
```

Comment:
I think this would be better defined in variables.scss as border-radius-xl (or something like that)